src/npcmove.cpp

```diff
@@ -3523,23 +3525,26 @@ void npc::find_item()
     }
 
     const auto consider_item =
-        [&wanted, &best_value, this]
+        [&best_value, this]
     ( const item & it, const tripoint & p ) {
         if( ::good_for_pickup( it, *this, p ) ) {
             wanted_item_pos = p;
-            wanted_item = &it;
-            wanted = wanted_item;
+            //wanted_item = &it;
+            //wanted = wanted_item;
```

Comment:
This is dead code now, don't comment it out. You'll confuse us simple folk when we come to edit this code in seven years time.